website/app/main.js

```diff
@@ -18,7 +18,7 @@ window.MacMessenger = {
   showSettings: function() {
     this.openGearMenu();
     document.querySelector(
-      'div.uiLayer.uiContextualLayerPositioner#js_1 > div.uiContextualLayer a'
+      'div.uiLayer.uiContextualLayerPositioner#js_2 > div.uiContextualLayer a'
```

Comment:
Element ids are dynamic (e.g. for me the id is currently `js_2z`). It's the same with data-reactid. And classes will also most likely change on update.